src/tests/system/tests/test_netgroups.py

```diff
@@ -309,3 +312,106 @@ def test_netgroup__uid_gt_2147483647(client: Client, provider: GenericProvider):
         result = client.tools.getent.group(grpname)
         assert result is not None, f"getent group for group '{grpname}' is empty!"
         assert result.name == grpname, f"Group name '{grpname}' did not match result '{result.name}'!"
+
```

Comment:
This is already covered and exists in test_cache.py 

https://github.com/SSSD/sssd/blob/9b6d8fe722c36419e01ee92faa8c54719df00623/src/tests/system/tests/test_cache.py#L28

Reply:
ok . this test is removed from this PR